modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/alert-info/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const messageReceived = localStorage.getItem('userAccountData');
+const data = JSON.parse(messageReceived);
+
+const loggedUser = document.querySelector('.logged-user');
+loggedUser.textContent = data.userName;
```

Comment:
This code will throw an error if messageReceived is empty (null/undefined).

Same applies to loggedUser element.

```suggestion
const messageReceived = localStorage.getItem('userAccountData');

if (messageReceived) {
    const data = JSON.parse(messageReceived);

    const loggedUserContainer = document.querySelector('.logged-user');
    
    if (loggedUserContainer) {
        loggedUser.textContent = data.userName;
    }
}
```

Reply:
Done